revengine/settings/deploy.py

```
@@ -10,29 +9,29 @@
 
 #### Critical settings
 
-SECRET_KEY = os.getenv("DJANGO_SECRET_KEY")
+SECRET_KEY = env.str("DJANGO_SECRET_KEY", "")
 
 ### Environment-specific settings
-ALLOWED_HOSTS = os.getenv("ALLOWED_HOSTS", "localhost").split(":")
+ALLOWED_HOSTS = env.list("ALLOWED_HOSTS", "localhost", delimeter=":")
 
 ## Email
 EMAIL_BACKEND = "anymail.backends.mailgun.EmailBackend"
 MAILGUN_SENDER_DOMAIN = "fundjournalism.org"
 
 ANYMAIL = {
-    "MAILGUN_API_KEY": os.getenv("MAILGUN_API_KEY", ""),
+    "MAILGUN_API_KEY": env.str("MAILGUN_API_KEY", ""),
     "MAILGUN_SENDER_DOMAIN": MAILGUN_SENDER_DOMAIN,
 }
 
 EMAIL_SUBJECT_PREFIX = f"[revengine {ENVIRONMENT.title()}] "
-DEFAULT_FROM_EMAIL = f"noreply@{os.getenv('DOMAIN', 'example.com')}"
+DEFAULT_FROM_EMAIL = f"noreply@{env.str('DOMAIN', 'example.com')}"
```

This is redundant. Have same line in base.py, which is imported here.

Also, thinking about prod in particular, wouldn't we want to not have a default here and raise an error if DOMAIN not defined, because I don't think we'd want to have emails being sent with from email that has a fake domain.